<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtkColorTable</class>
 <widget class="QWidget" name="VtkColorTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="mainTable">
     <column>
      <property name="text">
       <string>Scalar Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Red</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Green</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Blue</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QCheckBox" name="clampCheckBox">
       <property name="text">
        <string>Clamping</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="newDelSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
